apkid/rules/dex/obfuscators.yara

```diff
@@ -109,3 +109,29 @@ rule bitwise_antiskid : obfuscator
     is_dex and
     any of them
 }
+
+
+rule arxan : obfuscator
+{
+  meta:
+    description = "Arxan"
+    info        = "https://www.arxan.com/products/application-protection-mobile/"
+    sample      = "7bd1139b5f860d48e0c35a3f117f980564f45c177a6ef480588b5b5c8165f47e"
+
+  strings:
+    $obf_pkg = /L[a-z]{6}\//
+
+    $obf_mtd_00 = { 10 62 (6? | 75) [14] 00 }
```

Comment:
Nice work finding this. Could you add a little comment explaining what it is?

I know there's some discussion in the issue, but a little hint in the code is good in case it's separated from GitHub / the project.

Reply:
Thanks! It was @strazzere's idea but yeah, I managed to do it on my own and using Radare2.
Regarding explanation, there's a post talking about it. Many rules have no explanation whatsoever. Especially DexProtector or DexGuard. Only see opcodes. xDxD

If I find time, will do it.

Reply:
done!